organisaatio-ui/src/components/Modaalit/ToimipisteenYhdistys/TYBody.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useContext } from 'react';
```

Comment:
About filename: Is the "TY" prefix needed? These files are in dedicate module after all?

Reply:
This followed convention in the project for other components.